src/ProgressOnderwijsUtils.Drawing/ImageTools.cs

```diff
@@ -8,7 +7,7 @@
 
 //this is not recommended?? downsides are unclear though some suggest it might be slow.
 
-namespace ProgressOnderwijsUtils
+namespace ProgressOnderwijsUtils.Drawing

```

Comment:
Ivm string-searchability is mijn voorkeur om voor de naam ProgressOnderwijsUtilsDrawing boven ProgressOnderwijsUtils.Drawing.  Geen heeeele grooote voorkeur, dus als je een reden hebt om een punt te willen ben ik makkelijk over te halen ;-).

Reply:
Ik heb ook ergens een vage herinnering aan nadelen van `business.financieel` en `business.test` die we ook ooit hadden.  Weet jij daar nog iets van @markijbema ?  Iig, een unieke symbol lijkt me het veiligste.

Reply:
De reden die ik heb een punt te gebruiken is dat de rest van de wereld het zo doet.

Ben wel benieuwd naar concrete voorbeelden waarom dit slecht zou zijn, overigens.

Reply:
Het is wel wat kort door de bocht om te zeggen "de rest van de wereld doet het zo".  Er zijn uiteraard erg, erg veel projecten niet *niet* als namespace prefix een naam van een ander project hebben. Wat hebben ProgressOnderwijsUtils.Drawing en ProgressOnderwijsUtils nou echt met elkaar te maken?

Het string-search voordeel is dat als je een uniek token hebt, het gewoon heel makklijk is om daar betrouwbaar op te zoeken.  Dat kan niet als je token uit meerdere stukken bestaat verwijzingen naar de namespace dus niet noodzakelijkerwijs ooit de substring "ProgressOnderwijsUtils.Drawing" bevatten.

Reply:
We hadden toen al een business.financieel, en ik wilde een los project wat duidelijk onderscheidbaar was, dus heb dat busfinancieel genoemd. Was iig niet een vergelijkbare situatie.